src/batch_normalization.js

```diff
@@ -2,7 +2,7 @@
 
 import {add, sub, mul, div, pow} from './binary.js';
 import {reshape} from './reshape.js';
-import {Tensor} from './tensor.js';
+import {Tensor, Scalar} from './tensor.js';
```

Comment:
Thanks for introducing 'Scalar' class.

Please also update https://github.com/huningxin/webnn-baseline/blob/main/test/gemm_test.js#L13, thanks.
```js
        options.c = new Tensor([1], [C]); // L13
```

Reply:
Good catch. Thanks.

Reply:
Fixed it in the new commit. PTAL.

Reply:
LGTM, thanks.